src/test/java/seedu/address/model/person/EmailTest.java

```diff
@@ -60,16 +60,16 @@ public void isValidEmail() {
         assertFalse(Email.isValidEmail("peterjack@example.com.")); // domain name ends with a period
         assertFalse(Email.isValidEmail("peterjack@-example.com")); // domain name starts with a hyphen
         assertFalse(Email.isValidEmail("peterjack@example.com-")); // domain name ends with a hyphen
-        assertFalse(Email.isValidEmail("peterjack@example.c")); // top level domain has less than two chars
```

Comment:
Why is this test case removed? Does the changes to the regex make `isValidEmail("peterjack@example.c")` become true?